strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java

```diff
@@ -1,110 +0,0 @@
-package org.carlspring.strongbox.providers.repository;
```

Comment:
What is going on here? Why has this test class been removed?

Reply:
I haven't done any by hand changes in this file though...
Would it be possible that one of the changes in the other files can cause this?

Reply:
I don't know what you meant to do, but you will have to restore this file, which you seem to have accidentally deleted.

Reply:
Notepad++ .

Reply:
I would recommend that you checkout our master and start fresh. Just copy the changed files and re-apply them one by one carefully. You can reset your fork by the number of commits that you have made and sync it with our and then re-apply your changes on top of it.

Reply:
ok!i'll do that, thank you^^